features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogParser.java

```diff
@@ -141,9 +144,12 @@ protected static Date parseDate(final String dateString) {
             // Date pattern has been crearted and checked inside if loop instead of 
             // parsing date inside the exception class.
             if (dateString.matches(datePattern)) {
-                final DateFormat df = new SimpleDateFormat("yyyy-MM-dd", Locale.ROOT);
-                df.setTimeZone(TimeZone.getTimeZone("UTC"));
-                return df.parse(dateString);
+            	
+              LocalDate date = LocalDate.parse(dateString);
+              DateTimeFormatter df = DateTimeFormatter.ofPattern("yyyy-MM-dd",Locale.ROOT);
+              date.format(df);
+              return Date.from(date.atStartOfDay().atZone(ZoneOffset.UTC).toInstant());
```

Comment:
I think you can abbreviate this as:
Date.from(date.atStartOfDay(ZoneOffset.UTC).toInstant());